core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -596,11 +596,11 @@ private void checkExchangeRowCount(RelNode rel, double expected, double expected
   }
 
   @Test void testRowCountRightJoinEmptyFinite() {
-    final String sql = "select * from (select * from emp limit 0) as emp\n"
-        + "right join (select * from dept limit 4) as dept\n"
+    final String sql = "select * from (select * from emp limit 4) as emp\n"
+        + "right join (select * from dept limit 0) as dept\n"
         + "on emp.deptno = dept.deptno";
     checkRowCount(sql, 1D, // 0, rounded up to row count's minimum 1
-        0D, 4D); // 1 * 4
+        0D, 0D); // 0 * 4
```

Comment:
From the name of I am thinking this test intended to test `empty join right`? (now it is changed to test `left join empty`?

Reply:
You're right, I have reviewed the test and added more cases